### DISCUSSION TITLE
Can someone please help on this | Remix Routing

### DISCUSSION THREAD

**Discussion by ImAbhishekTomar** [remix-run/remix#2062] [Q&A] [answered]
Hi, 

I'm new to Remix and trying to learn.   
I am trying to build routing like this, so when I click on "Link Component 1" then Component one loads in the area & when I click on "Link Component 2" the Component 2 load.

https://remix.run/docs/en/v1/guides/routing I am trying with this like but it is not helping for my scenario. 

> here is my repo https://github.com/ImAbhishekTomar/remix-advance-template

![image](https://user-images.githubusercontent.com/6933841/155031095-adadb13e-12c0-4475-ac1d-13378fe7ca53.png)

Discord - https://discord.com/channels/770287896669978684/771068344320786452/944695061621387276

**Comment by na2hiro** [comment, marked as answer]
You're using `{props.children}` to render in the "spot", but you should use `<Outlet />` instead. You can read about it on the docs:

* About `<Outlet />` itself: In the docs of React Router, which Remix internally uses https://remix.run/docs/en/v1/other-api/react-router
* About real examples: Tutorials like "Developer Blog" https://remix.run/docs/en/v1/tutorials/blog

It's unfortunate that the [Routing doc page](https://remix.run/docs/en/v1/guides/routing) you're referencing is missing `<Outlet />` and other important images (it seems). I hope the page will get revised soon! cc @kentcdodds

**Reply by kentcdodds** [reply]
Noted! We need to improve that page a bit, but I added a quick reference to `Outlet` there for now. Thanks!
